src/components/app/exporters/YnabExporter.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <v-form
+    ref="vForm"
+    v-model="validated"
+  >
+    <v-checkbox
+      v-model="exporter.active"
+      label="Active"
+      hide-details="auto"
+      @change="changed = true"
```

Comment:
```suggestion
      label="Active"
      @change="changed = true"
```
![image](https://user-images.githubusercontent.com/17686879/93917297-79ecf600-fd13-11ea-9696-ceccb21c2a2b.png)